### ISSUE TITLE
resetTo screen from a tab replace this tab. Push screen has no effect

### ISSUE THREAD

**Issue by Colmea** [wix/react-native-navigation#690]
### Issue Description

Hi all,

I have some issues with navigator in a tab screen.
I listen to deep link in a specific screen which is a tab (because it's the only screen loaded and I don't know how/where to create a Root component to handle deep links).

When a deep link is triggered in this screen tab, I try to *resetTo* or *push*.

The *resetTo* function properly goes to the new screen, but **replace** the tab screen. Which means if I go to this tab again in the future, it will be the new screen. I don't understand why and how to avoid this ?

The *push* screen doesn't work. Is it a know issue ?

In other words, how to push a new screen (I don't want a modal) from a screen tab ?

Thanks for your help !

### Environment
* React Native Navigation version: 2.0.0-experimental.171
* React Native version: 0.38.1
* Platform(s) (iOS, Android, or both?): Android
* Device info (Simulator/Device? OS version? Debug/Release?): Devie Huawei P9 Lite. Android 6.0. Debug

**Comment by guyca** [comment]
Hey @Colmea 
I understand you're handling the deeplink in the root screen of one of your bottom tabs. Is that right?
Bottom tabs are not unmounted when you navigate away from them, meaning it's possible you're trying to push into a screen stack that's currently not visible. Could this be the case?

**Comment by chrise86** [comment]
@guyca what is the best way of handling deep linking in tabs? Is it to handle it all in 1 tab root screen? what about when the app is opened via a notification?

**Comment by HoogsterInc** [comment]
This is still a problem for me.
